package.json

```diff
@@ -85,7 +85,7 @@
     "tslib": "^2.3.0"
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx}": "eslint --cache --fix",
-    "*.{js,jsx,ts,tsx,css,md}": "prettier --write"
+    "*.{ts,tsx}": "eslint --cache --fix",
+    "*.{ts,tsx,css,md}": "prettier --write"
```

Comment:
updating next.config.js was causing some involved eslint parser errors, and since most of the high turnover code in this project is ts I feel less bad about disabling this to address the issue. :/